### DISCUSSION TITLE
Scan button in Open-Webui gone?

### DISCUSSION THREAD

**Discussion by stevenbaert** [open-webui/open-webui#6013] [Q&A] [answered]
Hi,

I might be overlooking something, but it seems like the “Scan Documents” button (RAG) is no longer available. Could you let me know what has replaced it?

My objective is to automatically scan for documents in a specific folder (synced via OneDrive) and check for updates at regular intervals, without having to manually upload files.

Thanks!
S

**Comment by cimichaelm** [comment]
I also noticed this.
It seems there was a recent change in commit 79c005a041b1ec2d39d1fb536666c2530858d31c

**Comment by tjbck** [comment, marked as answer]
`DOCS_DIR` env var has been deprecated in favour of **direct directory sync from the UI** and **API uploads**, to prevent issues people were facing with the webui being unresponsive when syncing `DOCS_DIR`.

![image](https://github.com/user-attachments/assets/d6f0aaf6-37c4-4cff-8620-5360f92c0a73)


More details on how to upload files using API: https://docs.openwebui.com/api/#-retrieval-augmented-generation-rag

**Reply by stevenbaert** [reply]
I based my design on this docs folder (synced it in backend to a Onedrive). So it's  just depricated like that?
Does not feel comfortable using a product (Open Source yes that is appreciated) which brings down features you actively use. 

So how do I rebuild the same approach then (didn't read docs yet)? Iow have a daily check of changes in docs on specific location?

**Reply by MiroslawskiMichal** [reply]
> `DOCS_DIR` env var has been deprecated in favour of **direct directory sync from the UI** and **API uploads**, to prevent issues people were facing with the webui being unresponsive when syncing `DOCS_DIR`.
> 
> ![image](https://private-user-images.githubusercontent.com/25473318/375109049-d6f0aaf6-37c4-4cff-8620-5360f92c0a73.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg3MTY0NzMsIm5iZiI6MTcyODcxNjE3MywicGF0aCI6Ii8yNTQ3MzMxOC8zNzUxMDkwNDktZDZmMGFhZjYtMzdjNC00Y2ZmLTg2MjAtNTM2MGY5MmMwYTczLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDEyVDA2NTYxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQxMDAxMWYzY2M3ZDMyY2FkOTBkZjJkNTMwZmQ1M2JhMjQ0YjBhY2Q2ZmFlMjc1YjViM2U4ZjlmYWFhZWE2YjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.3Z4IeiJWOjt9X-lSKu0mWBpxiCoRCZ_QCb1Alq6u5nA)
> 
> More details on how to upload files using API: https://docs.openwebui.com/api/#-retrieval-augmented-generation-rag

while trying to use api as mentioned in documentation I get http 307 and I don't see any uploaded files. DO you have any suggestions?
